docs/source/platform/schema-registry.md

```diff
@@ -1,102 +1,102 @@
+- `apollo service:push`: upload a new schema to the registry
+- `apollo service:download`: download a schema from the registry
+- `apollo service:check`: compare the local schema against running traffic and validate if proposed changes will break any live queries
 
-1. Starting up the service with an environment variable called `ENGINE_SCHEMA_TAG` will link metrics sent to Engine with the value of that environment variable. This is the best way to associate metrics so that the schema tag isn't hardcoded into the server.
-2. Alternatively, schema tag can be set within the `engine` settings of Apollo Server 2.2 and up:
+Type `apollo service --help` for full details on the commands available in the CLI.
 
-```js
-const server = new ApolloServer({
-  // rest of normal server settings
-  engine: {
-    schemaTag: 'beta',
-  },
-});
-```
+<h2 id="push">Uploading a schema</h2>
 
-Both the new version of the schema, as well as its performance and error metrics can be viewed using [Engine](https://engine.apollographql.com) and can even be used with [schema validation](./schema-validation.html).
+You publish a schema to the registry by running `apollo service:push` from within your respository. The CLI will know where to fetch your local schema from based on the configuration you gave in your `apollo.config.js` file. Each time a new veresion of your schema is published, it is logged in your schema history and it becomes the basis of comparison for `apollo service:check`.
 
-<h2 id="history">Schema history</h2>
+Here's what running `apollo service:push` will look like:
 
-As your schema grows and evolves to meet the needs of your product, it is helpful to see a history of changes for a team. This allows everyone to know when new features were introduced, when old fields were removed, and even link back to the commit that caused the change. Apollo Engine provides all the tooling needed to track this history in a simple way. Every time your schema is updated, you can simply run the [`apollo service:push`](#publish) command to keep an up to date history of your schema.
+```
+~/Development/apollo/example$ apollo service:push
+  ✔ Loading Apollo Project
+  ✔ Uploading service to Engine
 
-Each time a schema is published, it becomes the basis for comparison for validating future schemas and avoiding breaking changes. Therefore, a service should be pushed to [Engine](https://engine.apollographql.com) each time a new schema is deployed.
+id      schema        tag
+──────  ────────────  ───────
+190330  example-4218  staging
+```
 
-This is best accomplished from automatic steps within a continuous integration workflow and an example CircleCI configuration is available below.
+### Hooking into CI
```

Comment:
I think we need to mention that you need to hook into CI much earlier, not sure where.